packages/calling-component-bindings/src/utils/callUtils.ts

```diff
@@ -75,3 +83,31 @@ const memoizedUpdateDisplayName = memoizeFnAll((participantId: string, participa
     return participant;
   }
 });
+
+/**
+ * @private
+ */
+export const isTeamsCallParticipants = (
+  participants: CommunicationIdentifier[]
+): participants is (PhoneNumberIdentifier | MicrosoftTeamsUserIdentifier | UnknownIdentifier)[] => {
+  for (const p of participants) {
+    if (isCommunicationUserIdentifier(p)) {
+      return false;
+    }
+  }
+  return true;
+};
+
+/**
+ * @private
+ */
+export const isNonTeamsCallParticipants = (
+  participants: CommunicationIdentifier[]
```

Comment:
Would be helpful to have a comment and some unit tests

nit. Prefer to avoid having negative names as functions.. just because when someone starts doing !isNonTeamsCallParticipants.. It is really easy to mess up the logic.